core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43.java

```diff
@@ -60,10 +61,10 @@ public void run() {
                 PoolingHttpClientConnectionManager cm = new PoolingHttpClientConnectionManager();
                 cm.setMaxTotal(DEFAULT_MAX_TOTAL_CONNECTIONS);
                 cm.setDefaultMaxPerRoute(DEFAULT_MAX_CONNECTIONS_PER_ROUTE);
-                cm.setValidateAfterInactivity(REQUEST_TIMEOUT_IN_MILLIS);
 
                 httpClientRef.compareAndSet(null, HttpClients.custom()
                         .setConnectionManager(cm)
+                        .setConnectionReuseStrategy(new NoConnectionReuseStrategy())
```

Comment:
This seems to be bit bizzard to me on high level it appears that connection will be recreated every time and this might impact performance. I haven't dig deep though.

Reply:
I was letting the tests run with this option to see if it helps, but it does not.